nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -450,6 +458,10 @@ in
       ''
         UsePAM yes
 
+        ${optionalString (cfg.banner != "") ''
+          Banner ${pkgs.writeText "ssh_banner" cfg.banner}
```

Comment:
Oh how I wish I could put structural `settings` options everywhere as described in NixOS/rfcs#42. I think it's not a problem here because openssh handles multiple assignments gracefully, but if it didn't this could break for people who set `Banner` in `extraConfig` themselves as described in https://github.com/NixOS/rfcs/pull/42#issuecomment-523137258